app/controllers/PromoLandingPage.scala

```diff
@@ -27,14 +27,17 @@ object PromoLandingPage extends Controller {
     (for {
       promotion <- tpBackend.promoService.findPromotion(promoCode)
       promotionWithLandingPage <- promotion.asDigipack
-      html <- if ((evaluateStarts && promotionWithLandingPage.starts.isAfterNow) || promotionWithLandingPage.expires.exists(_.isBeforeNow)) None else Some(views.html.promotion.landingPage(edition, catalog, promoCode, promotionWithLandingPage, Config.Zuora.paymentDelay, PegdownMarkdownRenderer))
+      html <- if ((evaluateStarts && promotionWithLandingPage.starts.isAfterNow) || promotionWithLandingPage.expires.exists(_.isBeforeNow))
+        None
+      else
+        Some(views.html.promotion.landingPage(edition, catalog.digipack.month, promoCode, promotionWithLandingPage, Config.Zuora.paymentDelay, PegdownMarkdownRenderer))
     } yield Ok(html)).getOrElse(Redirect("/digital" ? ("INTCMP" -> s"FROM_P_${promoCode.get}")))
   }
 
   def preview() = GoogleAuthenticatedStaffAction { implicit request =>
     Form(Forms.single("promoJson" -> Forms.text)).bindFromRequest().fold(_ => NotFound, { jsString =>
       Json.fromJson[AnyPromotion](Json.parse(jsString)).asOpt.flatMap(_.asDigipack)
-      .map(p => views.html.promotion.landingPage(edition, catalog, p.codes.headOption.getOrElse(PromoCode("")), p, Config.Zuora.paymentDelay, PegdownMarkdownRenderer))
+      .map(p => views.html.promotion.landingPage(edition, catalog.digipack.month, p.codes.headOption.getOrElse(PromoCode("")), p, Config.Zuora.paymentDelay, PegdownMarkdownRenderer))
```

Comment:
there's a lot on one line, perhaps we can `val` up some of the parameters to make it neater?